external/Directory.Build.props

```diff
@@ -1,10 +1,10 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
-    <NugetRuntimeIdentifier>None</NugetRuntimeIdentifier>
     <ContainsPackageReferences>true</ContainsPackageReferences>
     <IncludeVSTestReferences Condition="'$(IncludeVSTestReferences)' == ''">true</IncludeVSTestReferences>
     <!-- We need configuration-specific assets files. -->
```

Comment:
Do we still want to set RestoreOutputPath?

Reply:
I tried to remove it and could not do so and still get targetgroup-specific assets files. We need those as these projects change their references based on configuration.

Reply:
Yeah Ok. nuget generally sets this to MSBuildProjectExtensionsPath https://github.com/NuGet/NuGet.Client/blob/5032a2e592d61639e6c5aefb1d84e26e9c6d4cfb/src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets#L594 which is usually set to base intermediate path https://github.com/Microsoft/msbuild/blob/aec1703e63f3e32ac12dd6946ba94a2b37bded63/src/Tasks/Microsoft.Common.props#L50 so yet I guess we do need to set this unless we want to start doing one restore for these depprojs for all configurations.

Reply:
I could probably write a target that evaluated all configurations and reported the set of TFMs and RIDs used before doing the restore, but I'd prefer to avoid that complexity.  We also often have TFM:RID tuples where we don't want to fully expand the cross-product which NuGet doesn't support in a single restore.  The minor cost of multiple restores during the all-configurations build and having these properties set seems like an OK trade off to me.